metricbeat/module/prometheus/collector/collector.go

```diff
@@ -29,6 +28,7 @@ import (
 const (
 	defaultScheme = "http"
 	defaultPath   = "/metrics"
+	nanInf        = 9223372036854775808
```

Comment:
where did you find this constant? Isn't it public in the Prometheus client?

Reply:
Oops I used the hardcoded value instead... It should be something like `math.NaN()`. I will make the change! Thanks!